.gitignore

```diff
@@ -5,6 +5,7 @@ __pycache__/
 
 # C extensions
 *.so
+*.mp4
```

Comment:
In this position it looks like .mp4 is a C extension. Could you move it out from under that heading?

Reply:
resolved with [9cad6b9](https://github.com/EthoML/VAME/pull/3/commits/9cad6b9c23ab8dba7c5fa8d7c5fc288cee9bab35)